Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSource.java

```diff
@@ -152,16 +150,19 @@ private String getFaceImageUrl(Proxy proxy, CardDownloadData card, boolean isTok
             }
         } else {
             // BY CARD NUMBER
-            // localized and default
+            // localized and none
+            String cn= card.getCollectorId() ;
+            if( cn.endsWith( "*" )) cn= cn.substring(0 , cn.length() -1) +"★" ;
+            else if( cn.endsWith( "+" )) cn= cn.substring(0 , cn.length() -1) +"†" ;
+            else if( cn.endsWith( "Ph" )) cn= cn.substring(0 , cn.length() -2) +"Φ" ;
```

Comment:
Must move shared code with chars replace to helper method like prepareCardNumber(String cardNumber)